src/util/yaml.ts

```diff
@@ -0,0 +1,38 @@
+import {readFile, writeFile} from 'fs/promises';
+
+import * as YAML from 'js-yaml';
+
+import {ImpactYaml} from '../types/yaml';
```

Comment:
@narekhovhannisyan it's challenging to type the yaml right now, everything under graph is not typed and also under the top level config/initilize node is also not typed. Both those sections are completely free form, every impl will be different.

There might be more we can type later on but right now super hard to judge, keep things untyped for as long as possible.